packages/components/button/src/button.js

```diff
@@ -38,9 +38,15 @@ export class TSButton extends TSElement {
 		return colorBackgroundTypes.indexOf(this.type) > -1 ? 'inverted' : 'default';
 	}
 
+	clickHandler(event) {
+		if (!this.disabled) {
```

Comment:
Just for clarity, please add an explicit check for the busy state.

Reply:
done! 👍